src/WebJobs.Script.WebHost/Security/IScriptSecretSerializer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    internal interface IScriptSecretSerializer
+    {
+        int SupportedFormatVersion { get; }
```

Comment:
Only introduced this here because we're making this changes in a backwards compatible way. The version lets us efficiently decide which serializer to use.